### ISSUE TITLE
The new MetadataAwareNameConverter is not used on Api-Platform

### ISSUE THREAD

**Issue by Soullivaneuh** [api-platform/core#2342]
Formerly https://github.com/api-platform/core/issues/2332#issuecomment-440668720

The name converted set on framework bundle is not passed directly on the normalizer but on the metadata aware name converter: https://github.com/symfony/symfony/pull/28505/files#diff-0e793081ceb720201745c982a568903fL1366

But, in api-platform, you still copy the "old" method for your normalizers: https://github.com/api-platform/core/blob/d67848110778af711c5f54226fe1234ec5668230/src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php#L77-L79

IMHO, the `name_converter` option should be deprecated (or is it used for something else?) and the `serializer.name_converter.metadata_aware` should be passed if exists.

The current workaround of this issue is quite simple, just add this line on the `api_platform` configuration:

```yaml
api_platform:
    name_converter: serializer.name_converter.metadata_aware
```

WDYT?

**Comment by dunglas** [comment]
This one is fixed, right?

**Comment by Soullivaneuh** [comment]
I fixed #2343, not this one AFAIK.

**Comment by norkunas** [comment]
Also encountered this problem now..
